api/RTCStatsReport.json

```
@@ -255,9 +258,8606 @@
+                "version_added": "≤80"
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "13.1> ≤14.1"
+              },
+              "safari_ios": {
+                "version_added": "≤16.4"
+              },
+              "samsunginternet_android": {
+                "version_added": "≤20.0"
+              },
+              "webview_android": {
+                "version_added": "≤112"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "dtlsRole": {
+          "__compat": {
+            "description": "<code>dtlsRole</code> in 'transport' stats",
+            "spec_url": "https://w3c.github.io/webrtc-stats/#dom-rtctransportstats-dtlsrole",
+            "support": {
+              "chrome": {
+                "version_added": "103"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "dtlsState": {
+          "__compat": {
+            "description": "<code>dtlsState</code> in 'transport' stats",
+            "spec_url": "https://w3c.github.io/webrtc-stats/#dom-rtctransportstats-dtlsstate",
+            "support": {
+              "chrome": {
+                "version_added": "≤80"
+              },
+              "chrome_android": {
+                "version_added": "≤112"
+              },
+              "edge": {
+                "version_added": "≤80"
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "13.1> ≤14.1"
```

Not in v13 according to https://bugs.webkit.org/show_bug.cgi?id=212668, so perhaps we set at 14.1?

```suggestion
                "version_added": "14.1"
```